iam/pgiam.py

```diff
@@ -383,7 +385,14 @@ def group_members(
         dict
 
         """
-        q = "select group_members('{0}')".format(group_name)
+        memberships_filtered = "true" if filter_memberships else "false"
+        memberships_filtered = "true" if client_timestamp else memberships_filtered
```

Comment:
can probably remove `memberships_filtered = "true" if client_timestamp else memberships_filtered`

Reply:
You cannot get filtered memberships without also providing `client_timestamp` then. Which I believe you may want to, i.e. filtered with default timestamp which is current time.

Reply:
Ah yes, how about `"true" if (filter_memberships and client_timestamp) else "false"`

Reply:
Must be `or` I believe, i.e. `"true" if (filter_memberships or client_timestamp) else "false"`.

Reply:
Have change to this.